Remotion/Web/Development.WebTesting/Configuration/WebTestConfigurationSection.cs

```diff
@@ -190,6 +197,11 @@ public string LogsDirectory
       get { return (string) this [_logsDirectoryProperty]; }
     }
 
+    public ChromiumDisableSecurityWarningsBehavior ChromiumDisableSecurityWarningsBehavior

```

Comment:
We need to decide on the inline vs. nested property. Is any of the other settings sepecifc to Chrome?

Reply:
No, there is no other Chrome or Chromium specific setting, which is why I wouldn't make is nested at this point.

Reply:
So we don't acutally have a precedence. How about a discussion on the pro and cos of nesting/not-nesting?